aks/terraform.tfvars.json

```diff
@@ -1,15 +1,15 @@
 {
   "location": "xxxxxxxxxxxxxxxxxxxxxxxxx",
-  "az_resource_group": "xxxxxxxxxxxxxxxxxxxxxxxxx",
+  "resource_group": "xxxxxxxxxxxxxxxxxxxxxxxxx",
   "ssh_public_key": "/xxxxxxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxxxxxxxxxxxxx.pub",
-  "agent_admin": "xxxxxxxxxxxxxxxxxxxxxxxxx",
+  "ssh_username": "xxxxxxxxxxxxxxxxxxxxxxxxx",
   "client_id": "xxxxxxxxxxxxxxxxxxxxxxxxx",
   "client_secret": "xxxxxxxxxxxxxxxxxxxxxxxxx",
   "cluster_labels": {},
   "azure_dns_json": "/xxxxxxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxxxxxxxxxxxxx/azure_dns_json.json",
```

Comment:
You probably don't want to source the `external-dns` azure config file from the filesystem. Instead, you can provide them to the helm deployment for external-dns as documented here:

https://github.com/helm/charts/tree/master/stable/external-dns

(check the azure parameters for resource group, tenant id, subscription id etc.)

Reply:
This file is cruft; removed in 1e04545 .